jena-jdbc/jena-jdbc-core/pom.xml

```diff
@@ -39,17 +39,16 @@
 			<artifactId>jena-arq</artifactId>
 			<version>4.0.0-SNAPSHOT</version>
 		</dependency>
-
-    <!-- Jena Driver configures Log4j directly -->
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
-    </dependency>
-
+    
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
-   </dependency>
+        <groupId>org.apache.logging.log4j</groupId>
+        <artifactId>log4j-slf4j-impl</artifactId>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Is this needed? (log4j2 has a BOM).

Reply:
Good catch, this gets implied by the `log4j-slf4j-impl` dependency